### ISSUE TITLE
Loss variance between validation and test

### ISSUE THREAD

**Issue by winddori2002** [facebookresearch/denoiser#70]
Hi,

I recently reproduced your works to use my team project.
I set the validation set as 286, 287 speakers as you mentioned in the previous issue.
However, there is a bit of loss variance between validation and test set when I used lightened denoiser.
The PESQ score on the test set is similar to your works, but the validation loss is much higher than the test set.

I'm curious whether I did something wrong.
Or is this the problem of the original dataset?
I've seen that the original train dataset contains the worst case for robust learning.

**Comment by adiyoss** [comment]
Hi @winddori2002, 
Do you mean the L1 loss or PESQ? What is the PESQ gap you observe between the test and val? What model are you using? the 48 or 64 version? 
If I remember correctly the gap in terms of PESQ was something like ~0.2 but I'll double check it to be sure.

**Comment by winddori2002** [comment]
Hi @adiyoss 

I use the 48 version and PESQ score is 2.7 on the test set and 2.3 on the valid set. 
Unlike your settings on paper, I train the model 70 epochs. 
I wonder that the PESQ score on the test set is better than the valid set.
Thank you
